### PR TITLE
[Display_02] DisplayCommInterface Test and Implementation

### PR DESCRIPTION
**Pull request by Dbaker19972**
Working on #68

### CODE REVIEW

**Review by sydneyhill3901 — CHANGES_REQUESTED** [review]
Go ahead and make the style changes requested by the bot when you get the chance, and I'll take another look at it later.

**Comment by andrewdimmer** [comment]
### NOTE: DO NOT COMMIT THE `/bin/` FOLDER!

This effectively doubles the size of the repo and breaks a whole bunch of stuff. I've fixed it now, I just wanted to let everyone on this branch know that one of you will likely be prompted to commit it now that it is untracked again, but please don't.

File: src/main/java/edu/oakland/production/display02/DisplayCommInterfaceImplementation.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  private DisplayComm2WayInterface comm2Way;
+
+  /**
+  * Creates a DisplayCommInterfaceImplementation to receive the rfid.
+  * param DisplatComm2WayInterface: The class that DisplayCommInterface will pass the rfid to.
+  *
+  */
+  public DisplayCommInterfaceImplementation(DisplayComm2WayInterface displayComm2WayInterface) {
+    if (displayComm2WayInterface == null) {
+      throw new IllegalArgumentException("processor cannot be null");
```
Comment (andrewdimmer) [inline src/main/java/edu/oakland/production/display02/DisplayCommInterfaceImplementation.java]: This should be as follows, as it's not really the processor:
```suggestion
      throw new IllegalArgumentException("displayComm2WayInterface cannot be null");
```

**Review by andrewdimmer — APPROVED** [review]
One minor change then you're all set to merge!

File: src/main/java/edu/oakland/production/display02/DisplayCommInterface.java [hunk 2]
```diff
@@ -3,7 +3,5 @@
 import edu.oakland.helper.admin.TrackData;
 
 public interface DisplayCommInterface {
-
```
Comment (sydneyhill3901) [inline src/main/java/edu/oakland/production/display02/DisplayCommInterface.java:6]: Good to merge.

**Review by sydneyhill3901 — APPROVED** [review]

**Pull request merged**